src/ensembl/src/shared/hooks/useHover.tsx

```diff
@@ -44,7 +44,7 @@ export default function useHover<T extends HTMLElement>(): UseHoverType<T> {
       element.addEventListener('mouseenter', handleMouseEnter);
       element.addEventListener('mouseleave', handleMouseLeave);
       element.addEventListener('click', handleMouseLeave);
-      element.addEventListener('touchstart', handleTouch);
+      element.addEventListener('touchstart', handleTouch, { passive: true });
```

Comment:
This has been added in https://github.com/Ensembl/ensembl-client/pull/576; shouldn't be part of this PR